### PR TITLE
Updated README.md

### PR DESCRIPTION
**Pull request by aj11anuj**
#### What this PR does / why we need it:
Made some minute changes in `README.md` for reducing confusion among new-comers while reading it, by changing some words that can be misleading.

/kind documentation

### CODE REVIEW

File: release-team/role-handbooks/docs/README.md [hunk 1]
```diff
@@ -33,8 +33,8 @@ Responsibilities include:
 * Creating a dev branch used by contributors to target documentation updates for the upcoming release
 * Offering guidance to contributors about how to contribute new feature and enhancements documentation and working with contributors to modify existing docs to accurately represent any upcoming changes 
 * Providing weekly updates to the Release Team about the current state of release-bound docs
-* Mentoring Docs Lead Shadows throughout this process and empowering them with the knowledge needed to be future Docs Leads
-* Working with SIG-Docs to review documentation PRs according to the website [Style Guide](https://kubernetes.io/docs/contribute/style/style-guide/) to ensure quality 
+* Mentoring Docs Shadows throughout this process and empowering them with the knowledge needed to be future Docs Leads
+* Working with Shadows to review documentation related PRs according to the [style guide] of (https://kubernetes.io/docs/contribute/style/style-guide/) website to ensure quality
```
Comment (saschagrunert) [inline release-team/role-handbooks/docs/README.md:37]: Hey, the link is visually broken now:

screenshot

**Review by saschagrunert — CHANGES_REQUESTED** [review]

File: release-team/role-handbooks/docs/README.md (hunk 1 above)
Comment (aj11anuj) [reply release-team/role-handbooks/docs/README.md:37]: Uh-oh, my bad

**Comment by aj11anuj** [comment]
Done!

**Review by sandipanpanda — COMMENTED** [review]
Thanks for your contribution. Please amend the commit message to be more descriptive of the change so that looking at the git log, people can get an idea of what was done. There are many README.md files in this repo.

You can refer: 
https://www.w3schools.com/git/git_amend.asp
https://cbea.ms/git-commit/

/hold

**Pull request closed without merging**